tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/BalxRunFunctionPositiveTestCase.java

```diff
@@ -95,10 +98,28 @@ public void testMultipleParam() throws BallerinaTestException {
         outLogLeecher.waitForText(2000);
     }
 
+    @Test
+    public void testArgWithColons() throws BallerinaTestException {
+        String arg = "test arg";
+        String fileName = "test_entry_function_with_colons.bal";
+        String[] args = {"-o",
+                tempProjectDirTwo.toString().concat(File.separator).concat("test_entry_function_with_colons"),
+                (new File("src/test/resources/run/balx/complex")).getAbsolutePath().concat(File.separator)
+                        .concat(fileName)};
+        balClient.runMain("build", args, null, new String[0], new LogLeecher[0],
+                          tempProjectDirTwo.toString());
```

Comment:
Shall we merge this line with the previous so that it comes to one line as
"balClient.runMain("build", args, null, new String[0], new LogLeecher[0], tempProjectDirTwo.toString());"?